Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -904,6 +904,10 @@ private GraphicInputEventData HandlePointerUp(IMixedRealityPointer pointer)
 
             GraphicInputEventData graphicEventData;
             FocusProvider.TryGetSpecificPointerGraphicEventData(pointer, out graphicEventData);
+            
+            GameObject focusedObject = FocusProvider.GetFocusedObject(pointer);
```

Comment:
We should already have a valid focused object in the `HandleEvent` call, otherwise we're just covering up the problem, not fixing it.